server/controllers/profileController.js

```diff
@@ -103,10 +103,10 @@ export default class ProfileController {
    * @returns {object} - status, message and profile detail
    */
   static updateProfile(req, res) {
-    const body = ProfileController.updateReqBody(req, req.user);
+    const body = ProfileController.updateReqBody(req);
 
     return Profile.findOne({
-      where: { username: req.params.username },
+      where: { username: req.user.username },
```

Comment:
Getting profile of an authenticated user should be done from the decode token, not by the params. So I fix this error